### ISSUE TITLE
FadeTo not completing on Samsung devices with visibility enhancement "Remove animations" turned on

### ISSUE THREAD

**Issue by eichbaum** [dotnet/maui#16476]
### Description

When turning on accessibility feature "Remove animations" on Samsung devices, FadeTo is not completing.

Settings > Accessibility > Visibility enhancements > Remove animations = turn on

This issue exists in Xamarin.Forms as well and seems to be a regression (fixed in Xamarin.Forms 4.4.0):
https://github.com/xamarin/Xamarin.Forms/issues/5686
https://github.com/xamarin/Xamarin.Forms/issues/15743

### Steps to Reproduce

1. Create new .NET MAUI app project
2. add `await CounterBtn.FadeTo(0.5);` in `private async void OnCounterClicked(object sender, EventArgs e)`
3. turn on Settings > Accessibility > Visibility enhancements > Remove animations (Samsung device required)
4. run and tap "Click me" button
5. CounterBtn.Text is not updating

When tapping "Click me" again, the first called `await CounterBtn.FadeTo(0.5);` completes and the text updates. I guess this happens as FadeTo is completing as soon as another FadeTo on the same button is started.

Besides this the opacity of the button is not changed (should be 0.5).

### Link to public reproduction project repository

https://github.com/eichbaum/FadeToDemo

### Version with bug

7.0.86

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 13, One UI Core Version 5.1

### Did you find any workaround?

turn Settings > Accessibility > Visibility enhancements > Remove animations off

**Comment by awasilik** [comment]
Hi,
We encounter the same problem for all basic animations
TranslateTo, FadeTo etc

**Comment by XamlTest** [comment]
Verified this on Visual Studio Enterprise 17.8.0 Preview 1.0. Repro on Android 13.0-API33(8.0.100-preview.7.23376.3) with below Project:
[FadeToDemo.zip](https://github.com/dotnet/maui/files/12354463/FadeToDemo.zip)

![FadeTo](https://github.com/dotnet/maui/assets/97647558/eb5da88b-b507-4d55-868c-3ee2ffd8c756)

**Comment by hartez** [comment]
See also: #16511 and #16478

**Comment by tristanhollman** [comment]
Our app has some animations on a custom splash/start screen. But users who have animations disabled in their settings can't get past the splash screen due to this issue 😭. Reproducible on different devices (not just Samsung) and also on the emulator in VS2022. We're currently still on .NET 7, but it also occurs the latest preview of .NET 8.

While we're waiting for the fix in .NET MAUI, does anyone know of a workaround for this issue? (apart from removing the animations all together 😊). It would be enough if we could manually check if the animations are enabled on the device to get around this issue.

**Comment by awasilik** [comment]
Hey, I see the issue is closed, does that mean it's already fixed with next release of .NET Maui? :)

**Comment by hartez** [comment]
Yes, the fix should be in the next release.
